apps/reference/docs/guides/_components/project_setup.mdx

```diff
@@ -0,0 +1,66 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import UserManagementSQLTemplate from './user_management_quickstart_sql_template.md'
+
+## Project set up
```

Comment:
Hmm looks like the table of contents isn't playing nicely with these partials/components:
![Screen Shot 2022-10-20 at 11 45 46 AM](https://user-images.githubusercontent.com/7026076/197032232-76759a6e-d2ac-4bf7-aa17-f235ffe6349e.png)

What headings should be included:
![Screen Shot 2022-10-20 at 11 47 04 AM](https://user-images.githubusercontent.com/7026076/197032438-159ff7a5-fd23-4cf0-b7c0-95feea7ea175.png)

Reply:
@dannykng any idea how to fix that? I'd prefer not to pull out the headlines. I'd say let's not block on this and fix in a separate PR!

Reply:
the only way to fix it is to keep the Headings in the page, and then just the content in the partial

I think the partials also mess up the navigation a bit (eg, you can "navigate" to the partial like it's an actual doc)